unit-tests/live/options/test-timestamp-domain.py

```diff
@@ -0,0 +1,90 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+# test: device D400*
+
+import time
+import pyrealsense2 as rs
+from rspy import test
+
+# This test checks Timestamp domain toggle option for color and depth sensor
+
+
+def frame_callback(frame):
+    global expected_time_stamp_domain
+    test.check_equal(int(frame.get_frame_timestamp_domain()), expected_time_stamp_domain)
+
+
+################################################################################################
+test.start("Start test")
+
+ENABLE = 1
+DISABLE = 0
+
+devices = test.find_first_device_or_exit()
+
+time.sleep(3)  # The device starts at D0 (Operational) state, allow time for it to get into idle state
+depth_sensor = devices.first_depth_sensor()
+color_sensor = devices.first_color_sensor()
+
+depth_sensor.get_option(rs.option.global_time_enabled)
```

Comment:
This 2 lines does nothing, ignoring the return value

Reply:
Absolutely. Lines was deleted.